src/DIRAC/DataManagementSystem/Client/FileCatalogClientCLI.py

```diff
@@ -1,17 +1,12 @@
 #!/usr/bin/env python
 """ File Catalog Client Command Line Interface. """
-# TODO: This should be modernised to use subprocess(32)
-try:
-    import commands
-except ImportError:
-    # Python 3's subprocess module contains a compatibility layer
-    import subprocess as commands
+import subprocess as commands
```

Comment:
No need to still name it `commands`
```suggestion
import subprocess
```